gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStorageFileAttributeView.java

```diff
@@ -59,19 +62,19 @@ public void setTimes(FileTime lastModifiedTime, FileTime lastAccessTime, FileTim
   public boolean equals(@Nullable Object other) {
```

Comment:
Aren't there multiple things in Maven that implement the JSR-305 annotations? Because if so, it might be a good idea for us to remove them entirely.